src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/config/services.yml

```diff
@@ -143,6 +143,12 @@ services:
     tags:
       - { name: tactician.handler, command: Surfnet\ServiceProviderDashboard\Application\Command\Entity\UpdateEntityAclCommand }
 
+  surfnet.dashboard.command_handler.update_entity_idps:
+    class: Surfnet\ServiceProviderDashboard\Application\CommandHandler\Entity\UpdateEntityIdpsCommandHandler
+    arguments:
+      $publishClient: '@Surfnet\ServiceProviderDashboard\Infrastructure\Manage\Client\PublishEntityClient'
+    tags:
+      - { name: tactician.handler, command: Surfnet\ServiceProviderDashboard\Application\Command\Entity\UpdateEntityIdpsCommand }
```

Comment:
This is not a priority to me, tactician is part of The PHP Leage, it should be supported for many years to come. I do not see a very strong benefit in moving to Symfonies Messenger. Do you see this differently?

Reply:
The thing that comes to mind is the wide spread of knowlegde. SF Messenger is already very common among symfony developers, so this would makes this easier to comprehend by other symfony developers.

Not a priority I think (it is actual a business decision as it currently fulfills its purpose)

Reply:
If I was to choose a pipeline right now, I would probably choose the Symfony Messenger. 

Thanks for bringing it up. Always nice to (re)consider choices of the past.